medcat/cdb.py

```diff
@@ -213,9 +213,9 @@ def add_names(self, cui: str, names: Dict, name_status: str = 'A', full_build: b
             # Name status must be one of the three
             name_status = 'A'
 
-        self.add_concept(cui=cui, names=names, ontologies=set(), name_status=name_status, type_ids=set(), description='', full_build=full_build)
+        self._add_concept(cui=cui, names=names, ontologies=set(), name_status=name_status, type_ids=set(), description='', full_build=full_build)
 
-    def add_concept(self,
```

Comment:
We want to keep the old method signature and annotate it with `medcat.utils.decorators.deprecated`.
Otherwise people who may be using this method somewhere down the line would have a breaking change.

Reply:
Added